### PR TITLE
Adds HTTPS logging endpoint support

### PR DESCRIPTION
**Pull request by mccurdyc**
## Relevant (Referenced) Link(s)

* [Fastly API Docs](https://developer.fastly.com/reference/api/logging/https/)
* [fastly/go-fastly](https://github.com/fastly/go-fastly/blob/c2c2c62210800daab641161412fbe210004aac0a/fastly/https.go)

## Validation

```
$ make test
```

```
git clone git@github.com:mccurdyc/cli.git /tmp/cli && \
(
    cd /tmp/cli && \
    git checkout mccurdyc/logging-https && \
    make test && \
    rm -rf /tmp/cli \
)
```

## Notes

* Assumes that https://github.com/fastly/cli/pull/90 gets merged first.

### CODE REVIEW

**Comment by mccurdyc** [comment]
Force-pushed rebasing into single commit.

File: pkg/logging/https/describe.go [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+	manifest manifest.Data
+	Input    fastly.GetHTTPSInput
+}
+
+// NewDescribeCommand returns a usable command registered under the parent.
+func NewDescribeCommand(parent common.Registerer, globals *config.Data) *DescribeCommand {
+	var c DescribeCommand
+	c.Globals = globals
+	c.manifest.File.Read(manifest.Filename)
+	c.CmdClause = parent.Command("describe", "Show detailed information about an HTTPS logging endpoint on a Fastly service version").Alias("get")
```
Comment (phamann) [inline pkg/logging/https/describe.go:26]:
```suggestion
	c.CmdClause = parent.Command("describe", "Show detailed information about a HTTPS logging endpoint on a Fastly service version").Alias("get")
```

File: pkg/logging/https/update.go [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+	ResponseCondition common.OptionalString
+}
+
+// NewUpdateCommand returns a usable command registered under the parent.
+func NewUpdateCommand(parent common.Registerer, globals *config.Data) *UpdateCommand {
+	var c UpdateCommand
+	c.Globals = globals
+	c.manifest.File.Read(manifest.Filename)
+
+	c.CmdClause = parent.Command("update", "Update an HTTPS logging endpoint on a Fastly service version")
```
Comment (phamann) [inline pkg/logging/https/update.go:50]:
```suggestion
	c.CmdClause = parent.Command("update", "Update a HTTPS logging endpoint on a Fastly service version")
```

**Comment by mccurdyc** [comment]
Force-pushed rebasing master.

**Comment by mccurdyc** [comment]
@phamann I think "a HTTPS" versus "an HTTPS" might be British English versus American English, respectively depending on pronunciation of the 'H'.

I _think_ we use American English elsewhere in the project and therefore propose sticking with "an HTTPS".

**Comment by mccurdyc** [comment]
@phamann The API docs use "an HTTPS", so to align with them, I'm going to leave it as is :)

https://developer.fastly.com/reference/api/logging/https/

**Pull request merged**